app/queue/queue.py

```diff
@@ -113,7 +113,7 @@ def add_host(host_data):
                     }
                 },
             )
-            (output_host, add_result) = host_repository.add_host(
+            (output_host, insights_id, add_result) = host_repository.add_host(
```

Comment:
We can retrieve Host ID here too so we don’t need to extract it from the serialized payload.

```suggestion
            output_host, host_id, insights_id, add_result = host_repository.add_host(
```